### PR TITLE
basic "proofreading"

### PR DESCRIPTION
**Pull request by baymoe**
fixed some grammatical and spelling errors, made the descriptions flow a little better

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@
 
 [![dockeri.co](https://dockeri.co/image/zixia/simple-mail-forwarder)](https://hub.docker.com/r/zixia/simple-mail-forwarder/)
 
-Simplest and Smallest Email Forward Service based on Docker.
+Simplest and Smallest Email Forward Service for Docker.
 
-1. Config by [**one-line**](#environment-variable-and-default)
+<!-- @Deprecated 1. Config by [**one-line**](#environment-variable-and-default) -->
```
Comment (huan) [inline README.md]: Cloud you please let me know why you suggest deprecate this line?

I believe this is still a very important feature for SMF.

File: README.md [hunk 2]
```diff
@@ -8,13 +8,13 @@
...
 1. Run as [**docker start**](#quick-start-tldr)
 1. Image Size [**10MB**](https://hub.docker.com/r/zixia/simple-mail-forwarder/tags/)
 
-View on Github - https://github.com/huan/docker-simple-mail-forwarder
+[View on Docker](https://hub.docker.com/r/zixia/simple-mail-forwarder)
```
Comment (huan) [inline README.md]: Thanks for adding the docker hub link.

However, please keep the GitHub link when you adding the DockerHub link.

Those two should all be displayed in our README.

File: README.md [hunk 3]
```diff
@@ -34,19 +34,19 @@ Voice from Users
...
 - [DuoCircle Email Forwarding](http://www.duocircle.com/services/email-forwarding) From $59.95/year
-- [Cloud Mail In](https://www.cloudmailin.com/plans) From $9/month. And it is not for human.
+- [Cloud Mail In](https://www.cloudmailin.com/plans) From $9/month. And it is not for humans.
 - [MailGun](https://mailgun.com) professional service. Free plan includes 10,000 emails/month. but [can result in your domain being treated as spam](https://blog.rajivm.net/mailgun-forwarding-spam.html)
 
-I was willing to pay $10/year, but the cheapest plan I could find was $9 per month. Having a $10 USD machine with unlimited mail&domains per month is an amazing idea! And of course you could also put other dockers on this same machine. :-D
+I was willing to pay $10/year, but the cheapest plan I could find was $9 per month. Having a $10 USD machine with unlimited e-mail & domains per month is an amazing idea! And of course you could also put other dockers on this machine. :-D
 
-Quick Start (TL;DR)
+Quick-Start (TL;DR)
```
Comment (huan) [inline README.md]: Quick-start

File: README.md (hunk 1 above)
Comment (baymoe) [reply README.md]: the reference link no longer works, the headline #environment-variable-and-default doesn't exist and i couldn't find where the "new" version is

File: README.md (hunk 2 above)
Comment (baymoe) [reply README.md]: I understand!

File: README.md (hunk 3 above)
Comment (baymoe) [reply README.md]: will change!

File: README.md (hunk 1 above)
Comment (huan) [reply README.md]: `#environment-variable-and-default-values`

I believe the reference link should be corrected to this: <https://github.com/huan/docker-simple-mail-forwarder#environment-variable-and-default-values>

**Comment by baymoe** [comment]
I fixed all your requests

**Comment by huan** [comment]
Thank you very much for the contribution.

Appreciate it!

**Pull request merged**